src/components/Books.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import BookItem from './BookItem';
+import Form from './Form';
+import '../styles/Books.css';
+
+function Books() {
+  return (
+    <div className="book-list">
+      <BookItem />
+      <BookItem />
+      <BookItem />
+      <Form />
+    </div>
```

Comment:
[REQUIRED] I don't think this is the best practice for your code. Let us assume you have an array of objects containing book data. Then you would do this:
```
 booksArray.map(book => {
return <Book book={book} />
}); 
```
You cannot keep writing out <BookItem/> especially when most times we are retrieving data from an external source. I am suggesting you do the following:
Create a <Books>` component, which contains the following components:

* `<BookList>`
  * Renders a list of books using the `<Book>` component
* `<Book>`
  * Takes the book data as a prop and renders the title and the author
  * Has a button for removing books
    * the button wouldn't remove the books *yet*, this is just a placeholder for now
`<NewBookForm>`

* Renders a form with inputs for the title and the author of the book
* Has a button to submit
  * the button wouldn't add the book *yet*, this is just a placeholder for now